### DISCUSSION TITLE
formily2修改样式前缀

### DISCUSSION THREAD

**Discussion by feixia5320** [alibaba/formily#1216] [Q&A 2.x]
由于我们工程中antd统一改了prefixcls，导致formily2中无法找到样式了，大神们有什么解决方法吗？

**Comment by janryWang** [comment]
1. 你的项目是用的antd3还是4，不支持3
2. 是否用的ConfigProvider，写法是怎样的

**Reply by feixia5320** [reply]
formily2只支持antd4以上版本，<ConfigProvider
      prefixCls={prefixCls}

**Reply by feixia5320** [reply]
需要引入formily的less文件，但是没找到入口less文件

**Reply by janryWang** [reply]
@formily/antd在index.ts内部会默认引入less文件的

**Reply by janryWang** [reply]
https://github.com/alibaba/formily/blob/formily_next/packages/antd/src/index.ts#L1

**Reply by feixia5320** [reply]
不用引入样式文件吗？不引入样式文件，不生效啊

**Reply by janryWang** [reply]
你改了前缀，应该webpack配置也得改的，因为要修改formily内部less文件的前缀

**Reply by scottjeremy** [reply]
@feixia5320 现在这个问题解决了吗？

**Reply by xiaobeiOfficial** [reply]
> @feixia5320 现在这个问题解决了吗？

同文，最终formily2.x是如何自定义样式前缀的，翻遍文档也没有看到。。我项目中加的自定义prefixCls是生效的，但是问题也是一样的，就是formily相关的样式全部丢失了

**Reply by janryWang** [reply]
你没显示加载样式的话是需要配置按需打包的

**Reply by xiaobeiOfficial** [reply]
> 你没显示加载样式的话是需要配置按需打包的

加了按需引入配置，现在OK了，thanks!
